extensions/ql-vscode/src/cli.ts

```diff
@@ -1255,7 +1255,7 @@ export class CliVersionConstraint {
   public static CLI_VERSION_WITH_NO_PRECOMPILE = new SemVer('2.7.1');
 
   /**
-   * CLI version where remote queries are supported.
+   * CLI version where remote queries (variant analysis) are supported.
```

Comment:
I think it's ok to assume that people reading the code know that remote queries = variant analysis (otherwise where we could end up adding `(variant analysis)` everywhere.